bears/vcs/git/GitCommitBear.py

```diff
@@ -31,6 +31,18 @@ class GitCommitBear(GlobalBear):
     }
     CONCATENATION_KEYWORDS = [r',', r'\sand\s']
 
+    _nltk_data_downloaded = False
+
+    @classmethod
+    def setup_dependencies(cls):
```

Comment:
This downloads a fairly large dataset (~200MB unzipped). If I'm understanding this correctly `setup_dependencies` is done for all runs of the bear. Ideally we would only want it to be run when `check_imperative` is selected by the user.